template/default.tmpl

```diff
@@ -123,6 +123,7 @@ Alerts Resolved:
 {{ end }}
 {{ end }}
 
+{{ define "discord.default.content" }}{{ end }}
```

Comment:
You want to update `DefaultDiscordConfig` on Line 46 of `config/notifiers.go` so this template is used.